src/ir/ssa_names.h

```diff
@@ -54,6 +59,7 @@ class SsaNames {
     absl::flat_hash_map<const T *, ID> item_ids_;
   };
 
+  IDManager<Module> module_ids_;
```

Comment:
Nit: It looks like you generally use the order {`Block`, `Operation`, `Module`} in this file, but here you are using {`Module`, `Operation`, `Block`}. Move this to the end of the field decls to make this a bit more uniform?

Reply:
Chose `{Operation, Block, Module}` to match `module.h` and made this consistent.